### ISSUE TITLE
unknown node: graphviz

### ISSUE THREAD

**Issue by RiccardoManfrin** [sphinx-contrib/confluencebuilder#318]
This is similar to #304 

`
Exception occurred:
  File "/usr/local/lib/python3.8/site-packages/sphinxcontrib/confluencebuilder/translator.py", line 149, in unknown_visit
    raise NotImplementedError('unknown node: ' + node_name)
NotImplementedError: unknown node: graphviz
`

**Comment by adrianohrl** [comment]
I've tried to use the `sphinx.ext.graphviz` along with the `sphinxcontrib.confluencebuilder`; although, a `NotImplementedError` is prompted to me. The same as reported above.

Is there any update to this issue since its submission?

**Comment by adrianohrl** [comment]
As a temporary solution for this, I've attached the following snippet in the project `conf.py` file:

```python
from graphviz import Source, render


def convert() -> None:
    suffix = '.svg'
    path = Path('_static/graphs').resolve()
    for path in path.rglob('*.dot'):
        print(f'Reading the {path} file ...')
        graph = Source.from_file(path)
        print(f'Writing the {path}{suffix} file ...')
        render(path.suffix[1:], suffix[1:], path)


convert()
```

It converts the all `DOT` file in the `_static/graphs` folder to `SVG`. Then, while documenting, I can use the `image` and `figure` directives pointing to the converted files. 

> Note: You may include `**/*.dot.svg` to the project's `.gitignore` file.

**Comment by jdknight** [comment]
The changes outlined in #373 are now available on the main branch and will be available next stable release (v1.3+). Will mark as pending-next-release and will close up when the release has been completed (see #362 for more details).

**Comment by jdknight** [comment]
v1.3 is now available on PyPI -- marking as closed.
